src/fdb_kvdb.c

```diff
@@ -1076,7 +1076,7 @@ static uint32_t new_kv(fdb_kvdb_t db, kv_sec_info_t sector, size_t kv_size)
             already_gc = true;
             goto __retry;
         } else if (already_gc) {
-            FDB_INFO("Error: Alloc an KV (size %" PRIuLEAST16 ") failed after GC. KV full.\n", kv_size);
+            FDB_INFO("Error: Alloc an KV (size %" PRIuPTR ") failed after GC. KV full.\n", (uintptr_t)kv_size);
```

Comment:
```suggestion
            FDB_INFO("Error: Alloc an KV (size %" PRIu32 ") failed after GC. KV full.\n", (uint32_t)kv_size);
```
Is it possible to modify it like this?